applicationinsights.ts

```diff
@@ -320,10 +321,11 @@ export class Configuration {
      * @param value if true Web snippet will be tried to be injected in server response
```

Comment:
Add new @param and mention the default is to use same connectionString as Node.js app